team/requirements.txt

```diff
@@ -1,6 +1,48 @@
-Flask==0.10.1
-itsdangerous==0.24
-Jinja2==2.8
```

Comment:
Why are you reverting Jinja?

Reply:
O.K.

Reply:
This is the new jinja version - Jinja2==2.9.6, so it should be backward compatible.